mypy/fastparse.py

```diff
@@ -1535,6 +1535,12 @@ def visit_JoinedStr(self, n: ast3.JoinedStr) -> Expression:
         # Don't make unnecessary join call if there is only one str to join
         if len(strs_to_join.items) == 1:
             return self.set_line(strs_to_join.items[0], n)
+        elif len(strs_to_join.items) > 1:
+            last = strs_to_join.items[-1]
+            if isinstance(last, StrExpr) and last.value == "":
+                # 3.12 can add an empty literal at the end. Delete it for consistency
```

Comment:
Interesting, why is that needed in CPython 🤔

Reply:
> Interesting, why is that needed in CPython 🤔

It caused a crash over at flake8-bugbear too: https://github.com/PyCQA/flake8-bugbear/issues/393

Reply:
If this has caused issues in two projects, should we report upstream?

Reply:
Might be worth it, yeah